mne/viz/topo.py

```diff
@@ -757,10 +757,13 @@ def _plot_evoked_topo(evoked, layout=None, layout_scale=0.945,
             y_label.append('Amplitude (%s)' % unit)
 
     if ylim is None:
-        # find maxima over all evoked data for each channel pick
-        ymaxes = np.array([max(np.abs(e.data[t]).max() for e in evoked)
+        # find minima and maxima over all evoked data for each channel pick
+        ymaxes = np.array([max((e.data[t]).max() for e in evoked)
```

Comment:
do you need this list comprehension now? Can't you simply do:

```py
ymaxes = evoked.data.max(axis=1)
```

Reply:
I think we still need the list comprehension in the case of two-channel types being plotted (like mag and grad). Otherwise, this would be better.

Reply:
sorry I missed that, but you can still do:

```py
ymaxes = np.max(evoked.data[picks], axis=1)
```